src/C-DEngine/C-DEngines/C-DNMIService/cde_NMIAPI.cs

```diff
@@ -2901,7 +2905,7 @@ private static void SetStatusInfoButtons(TheThing pBaseThing, TheFieldInfo tFld,
             tFlds["LastMessage"] = TheNMIEngine.AddSmartControl(pBaseThing, tMyForm, eFieldType.TextArea, StartFldOrder + 3, 0, 0, null, "LastMessage", new nmiCtrlSingleEnded() { NoTE = true, ParentFld = pParentFld, TileWidth = 4, TileHeight = UseBigStatus ? 2 : 1 });
             tFlds["LastUpdate"] = TheNMIEngine.AddSmartControl(pBaseThing, tMyForm, eFieldType.DateTime, StartFldOrder + 4, 0, 0, "###Last Update###", "LastUpdate", new nmiCtrlDateTime() { ParentFld = pParentFld, TileFactorY = tTFY, TileWidth = 6, TileHeight = 1 });
             tFlds["Value"] = TheNMIEngine.AddSmartControl(pBaseThing, tMyForm, eFieldType.Number, StartFldOrder + 5, 0, 0xFE, "###Current Value###", "Value", new nmiCtrlNumber() { ParentFld = pParentFld, TileFactorY = tTFY, TileWidth = 6, TileHeight = 1 });
-            tFlds["NodeName"] = TheNMIEngine.AddSmartControl(pBaseThing, tMyForm, eFieldType.SingleEnded, StartFldOrder + 6, 0, 0xFE, "###Managed on Node###", null, new nmiCtrlSingleEnded() { DataItem="cdeN", TileFactorY = tTFY, ParentFld = pParentFld, TileWidth = 6, TileHeight = 1 });
+            tFlds["NodeName"] = TheNMIEngine.AddSmartControl(pBaseThing, tMyForm, eFieldType.SingleEnded, StartFldOrder + 6, 0, 0, "###Managed on Node###", null, new nmiCtrlSingleEnded() { DataItem="cdeN", TileFactorY = tTFY, ParentFld = pParentFld, TileWidth = 6, TileHeight = 1 });
```

Comment:
0xFE is correct here. Anonymous users should not see this column

Reply:
They see the info elsewhere already (category for many plug-ins, for example, Status tile), and it does improve usability. Let's go over the plug-in UX I'm working on and then decide together.